mobile/src/main/java/com/github/shadowsocks/GlobalSettingsPreferenceFragment.kt

```diff
@@ -34,6 +40,22 @@ import com.github.shadowsocks.utils.TcpFastOpen
 import com.takisoft.preferencex.PreferenceFragmentCompat
 
 class GlobalSettingsPreferenceFragment : PreferenceFragmentCompat() {
+    private lateinit var toolbar: Toolbar
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        toolbar = inflater.inflate(R.layout.toolbar_light_dark, container, false) as Toolbar
+        return super.onCreateView(inflater, container, savedInstanceState)
+    }
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        val child = (view as LinearLayout).getChildAt(0)
```

Comment:
This line assumes the implementation of preference fragment which is probably subject to change.